src/deepsparse/transformers/pipelines/text_generation.py

```diff
@@ -572,9 +574,11 @@ def engine_forward(
 
         :param engine_inputs: list of numpy inputs to
             Pipeline engine forward pass
-        :return: A tuple of numpy array that contains the
-            sequence of generated tokens and a sequence
-            of logits for each generated token
+        :return: A tuple of:
+            - numpy array that contains the sequence
+                of generated tokens
+            - numpy array that cointains the sequence of
```

Comment:
is this a tangential change?

Reply:
also looks out of date after rebase

Reply:
Oh yes, this is a tangential change, sorry, you are right. Yes, it is out of date.